spec/rubocop/cop/rspec/it_behaves_like_spec.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+
+RSpec.describe RuboCop::Cop::RSpec::ItBehavesLike, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:cop_config) do
+    { 'EnforcedStyle' => enforced_style }
+  end
+
+  shared_examples 'an it_behaves_like cop' do |preferred, alternate|
+    let(:enforced_style) { preferred }
+
+    it 'flags a violation for the alternate term' do
+      inclusion = "#{alternate} 'a foo'"
+      expect_violation(<<-RUBY)
+        #{inclusion}
+        #{'^' * inclusion.length} Prefer `#{preferred}` over `#{alternate}` when including examples in a nested context.
```

Comment:
If you are going to use a shared context then I'd just use `inspect_source`. Grep the other examples for that usage.

I'd actually prefer the duplication of just writing the violation twice in the different contexts